com.bixense.PasswordCalculator.json

```diff
@@ -0,0 +1,60 @@
+{
+	"app-id": "com.bixense.PasswordCalculator",
+	"runtime": "org.gnome.Platform",
```

Comment:
You probably want to use `org.freedesktop.Platform` `1.6` since this project doesn't seem to use anything GNOME specific.

Reply:
What about Gtk+?

Reply:
GTK2 and 3 are part of of Freedesktop runtime.

Reply:
Only Gtk3 is in fdo.